app/src/androidTest/java/com/github/gogetters/letsgo/activities/MapsActivityTest.kt

```diff
@@ -3,29 +3,42 @@ package com.github.gogetters.letsgo.activities
 import androidx.test.espresso.Espresso.onView
 import androidx.test.espresso.action.ViewActions.click
 import androidx.test.espresso.assertion.ViewAssertions.matches
-import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
-import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.RootMatchers.withDecorView
+import androidx.test.espresso.matcher.ViewMatchers.*
 import androidx.test.ext.junit.rules.ActivityScenarioRule
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import androidx.test.platform.app.InstrumentationRegistry
 import androidx.test.platform.app.InstrumentationRegistry.getInstrumentation
-import androidx.test.uiautomator.UiDevice
-import androidx.test.uiautomator.UiObject
-import androidx.test.uiautomator.UiSelector
+import androidx.test.uiautomator.*
 import com.github.gogetters.letsgo.R
-import com.github.gogetters.letsgo.map.mocking.MockLocationSharingService
+import com.github.gogetters.letsgo.database.Database
 import com.github.gogetters.letsgo.database.EmulatedFirebaseTest
+import com.google.android.gms.maps.model.LatLng
+import junit.framework.Assert.assertFalse
+import junit.framework.Assert.assertTrue
+import org.hamcrest.Matchers.not
 import org.junit.After
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
+import org.junit.rules.ExpectedException
 import org.junit.runner.RunWith
 
 
 @RunWith(AndroidJUnit4::class)
 class MapsActivityTest : EmulatedFirebaseTest() {
     val GRANT_PERMISSION_BUTTON_INDEX = 0
     val PERMISSIONS_DELAY = 5000L
+    private val PACKAGE_NAME = "com.github.gogetters.letsgo"
+
+    val userPath = "users"
+    val isActivePath = "/isLookingForPlayers"
+    val latPath = "/lastPositionLatitude"
+    val lngPath = "/lastPositionLongitude"
+    val EPFL: LatLng = LatLng(46.51899505106699, 6.563449219980816)
```

Comment:
maybe use a variable for those

Reply:
Naaa, I don't think thats necessary in a test..